lib/db/mem.js

```diff
@@ -613,14 +613,27 @@ module.exports = function (log, error) {
 
     var accountId = sessionTokens[id].uid.toString('hex')
     var account = accounts[accountId]
-    item.emailVerified = account.emailVerified
-    item.email = account.email
-    item.emailCode = account.emailCode
-    item.verifierSetAt = account.verifierSetAt
-    item.locale = account.locale
-    item.accountCreatedAt = account.createdAt
 
-    return P.resolve(item)
+    return this.accountEmails(accountId)
```

Comment:
Had to add this so that user gets correct email in session after they change it. Previously, `session.email` was always the email on the account table.